client/src/app/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { Inter } from "next/font/google";
+import styles from "./page.module.css";
```

Comment:
Oh cool! Are these styles generated by default or did you have to write these?

Reply:
This is generated by default with create-next-app